### PR TITLE
Usagi Yojimbo

### PR DESCRIPTION
**Pull request by depperm**
A fun comic that immerses the reader in fantasy japan with animal characters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -861,6 +861,12 @@ Kate DiCamillo takes us on an extraordinary journey, from the depths of the ocea
 >
 > And then, one day, he was lost.
 
+[Usagi Yojimbo](https://www.goodreads.com/series/71007-usagi-yojimbo) (1987) _by [Stan Sakai](https://en.wikipedia.org/wiki/Stan_Sakai)_ [4.1]
```
Comment (RichardLitt) [inline README.md:864]: Missing ###

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (depperm) [inline README.md:864]:
```suggestion
### [Usagi Yojimbo](https://www.goodreads.com/series/71007-usagi-yojimbo) (1987) _by [Stan Sakai](https://en.wikipedia.org/wiki/Stan_Sakai)_ [4.1]
```

File: README.md [hunk 2]
```diff
@@ -861,6 +861,12 @@ Kate DiCamillo takes us on an extraordinary journey, from the depths of the ocea
 >
 > And then, one day, he was lost.
 
+### [Usagi Yojimbo](https://www.goodreads.com/series/71007-usagi-yojimbo) (1987) _by [Stan Sakai](https://en.wikipedia.org/wiki/Stan_Sakai)_ [4.1]
```
Comment (depperm) [inline README.md:864]:
```suggestion
#### [Usagi Yojimbo](https://www.goodreads.com/series/71007-usagi-yojimbo) (1987) _by [Stan Sakai](https://en.wikipedia.org/wiki/Stan_Sakai)_ [4.1]
```

**Comment by RichardLitt** [comment]
Thanks.

**Pull request merged**